### PR TITLE
Laser gun fixes and extra description tidbits.

### PR DESCRIPTION
**Pull request by David-Stormwell**
## About The Pull Request

Needed to remove some white from the sniper, adjusted color on the lasgun 100% charge cell, and added some extra info some descriptions.

## Why It's Good For The Game

fixes and stuff good

## Changelog
:cl:
fix: fixed some icon states for the sniper, adjusted a color for the lasgun cell 100% charged icon, and added extra info descriptions of some guns, fixed a firerate issue with the laser MG
/:cl:

### CODE REVIEW

**Review by TiviPlus — APPROVED** [review]

**Comment by David-Stormwell** [comment]
hold on the merge need to add one more fix soon

**Comment by David-Stormwell** [comment]
Alright feel free to merge

**Comment by David-Stormwell** [comment]
merge when

**Comment by David-Stormwell** [comment]
@SplinterGP

File: code/modules/projectiles/guns/energy.dm [hunk 1]
```diff
@@ -885,7 +885,7 @@
 	wield_delay = 1 SECONDS
 	scatter = 0
 	scatter_unwielded = 10
-	fire_delay = 0.15 SECONDS
+	fire_delay = 0.2 SECONDS
```
Comment (TiviPlus) [inline code/modules/projectiles/guns/energy.dm:888]: Undocumented balance changes

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/guns/energy.dm (hunk 1 above)
Comment (David-Stormwell) [reply code/modules/projectiles/guns/energy.dm:888]: It isn't.
The firemode is 0.2 already, this was something I missed in the first PR.

**Pull request closed without merging**